<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1048</width>
    <height>569</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="settingsBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>220</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>220</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string>Settings</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="Cylinder">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <attribute name="title">
           <string>Cylinder</string>
          </attribute>
          <widget class="QFrame" name="cylinderBox">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>196</width>
             <height>141</height>
            </rect>
           </property>
           <widget class="QLabel" name="labelRadius">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>30</y>
              <width>54</width>
              <height>17</height>
             </rect>
            </property>
            <property name="text">
             <string>Radius</string>
            </property>
           </widget>
           <widget class="QLabel" name="labelAngle">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>70</y>
              <width>54</width>
              <height>17</height>
             </rect>
            </property>
            <property name="text">
             <string>Angle</string>
            </property>
           </widget>
           <widget class="QLabel" name="labelHeight">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>110</y>
              <width>54</width>
              <height>17</height>
             </rect>
            </property>
            <property name="text">
             <string>Height</string>
            </property>
           </widget>
           <widget class="QDoubleSpinBox" name="radiusSpinBox">
            <property name="geometry">
             <rect>
              <x>110</x>
              <y>30</y>
              <width>62</width>
              <height>26</height>
             </rect>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
            <property name="value">
             <double>0.500000000000000</double>
            </property>
           </widget>
           <widget class="QDoubleSpinBox" name="angleSpinBox">
            <property name="geometry">
             <rect>
              <x>110</x>
              <y>70</y>
              <width>62</width>
              <height>26</height>
             </rect>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
            <property name="value">
             <double>0.500000000000000</double>
            </property>
           </widget>
           <widget class="QDoubleSpinBox" name="heightSpinBox">
            <property name="geometry">
             <rect>
              <x>110</x>
              <y>110</y>
              <width>62</width>
              <height>26</height>
             </rect>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
            <property name="value">
             <double>2.000000000000000</double>
            </property>
           </widget>
          </widget>
          <widget class="QSlider" name="TimeSlider">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>170</y>
             <width>160</width>
             <height>16</height>
            </rect>
           </property>
           <property name="maximum">
            <number>49</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="Path">
          <attribute name="title">
           <string>Path</string>
          </attribute>
         </widget>
         <widget class="QWidget" name="tab_2">
          <attribute name="title">
           <string>Tab 2</string>
          </attribute>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="rotationBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>205</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>205</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="title">
          <string>Rotation</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QDial" name="RotationDialX">
              <property name="toolTip">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Rotation along the X-axis&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="maximum">
               <number>359</number>
              </property>
              <property name="wrapping">
               <bool>true</bool>
              </property>
              <property name="notchTarget">
               <double>12.000000000000000</double>
              </property>
              <property name="notchesVisible">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDial" name="RotationDialY">
              <property name="toolTip">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Rotation along the Y-axis&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="maximum">
               <number>359</number>
              </property>
              <property name="wrapping">
               <bool>true</bool>
              </property>
              <property name="notchTarget">
               <double>12.000000000000000</double>
              </property>
              <property name="notchesVisible">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDial" name="RotationDialZ">
              <property name="toolTip">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Rotation along the Z-axis&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="maximum">
               <number>359</number>
              </property>
              <property name="wrapping">
               <bool>true</bool>
              </property>
              <property name="notchTarget">
               <double>12.000000000000000</double>
              </property>
              <property name="notchesVisible">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="ResetRotationButton">
            <property name="text">
             <string>Reset Rotation</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="scalingBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>205</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>205</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="title">
          <string>Scaling</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QSlider" name="ScaleSlider">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>200</number>
            </property>
            <property name="value">
             <number>100</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="invertedAppearance">
             <bool>false</bool>
            </property>
            <property name="invertedControls">
             <bool>false</bool>
            </property>
            <property name="tickPosition">
             <enum>QSlider::NoTicks</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="ResetScaleButton">
            <property name="text">
             <string>Reset Scaling</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="MainView" name="mainView"/>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MainView</class>
   <extends>QOpenGLWidget</extends>
   <header location="global">mainview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
